.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
-  - sudo pip3 install -r requirements.txt
+  - sudo pip3 install -U -r requirements.txt
```

Comment:
I don't know why this is happening, but colorama is also required by radon(>=0.3,<0.4) and pylint, apart from PyPrint. Here is dependency tree by `pipdeptree` for coala-bears:
```
coala-bears==0.2.1
  - guess-language-spirit [required: ~=0.5.2, installed: 0.5.2]
  - proselint [required: ~=0.3.5, installed: 0.3.6]
    - future [installed: 0.15.2]
    - six [installed: 1.10.0]
    - click [installed: 6.6]
  - pylint [required: ~=1.5.2, installed: 1.5.5]
    - astroid [required: >=1.4.5,<1.5.0, installed: 1.4.5]
      - six [installed: 1.10.0]
      - wrapt [installed: 1.10.8]
      - lazy-object-proxy [installed: 1.2.2]
    - six [installed: 1.10.0]
    - colorama [installed: 0.3.7]
  - cmakelint [required: ~=1.3.4, installed: 1.3.4.1]
  - eradicate [required: ~=0.1.6, installed: 0.1.6]
  - yamllint [required: ~=1.0.1, installed: 1.0.4]
    - pyyaml [installed: 3.11]
  - munkres3 [required: ~=1.0.5.5, installed: 1.0.5.5]
  - appdirs [required: ~=1.4.0, installed: 1.4.0]
  - radon [required: ~=1.3.1, installed: 1.3.2]
    - colorama [required: >=0.3,<0.4, installed: 0.3.7]
    - mando [required: >=0.3,<0.4, installed: 0.3.3]
  - isort [required: ~=4.2.2, installed: 4.2.5]
  - cpplint [required: ~=0.0.7, installed: 0.0.9]
  - language-check [required: ~=0.7.5, installed: 0.7.6]
  - vim-vint [required: ~=0.3.6, installed: 0.3.6]
    - chardet [required: ==2.3.0, installed: 2.3.0]
    - ansicolor [required: ==0.2.4, installed: 0.2.4]
    - PyYAML [required: ==3.11, installed: 3.11]
  - autoflake [required: ~=0.6.6, installed: 0.6.6]
    - pyflakes [required: >=0.8.1, installed: 1.2.3]
  - nltk [required: ~=3.1.0, installed: 3.1]
  - setuptools [required: >=19.2]
  - libclang-py3 [required: ==0.2, installed: 0.2]
  - pydocstyle [required: ~=1.0.0, installed: 1.0.0]
  - requests-mock [required: ~=0.7.0, installed: 0.7.0]
    - requests [required: >=1.1, installed: 2.9.2]
    - six [installed: 1.10.0]
  - autopep8 [required: ~=1.2.1, installed: 1.2.4]
    - pep8 [required: >=1.5.7, installed: 1.7.0]
  - coala [required: ~=0.6.0, installed: 0.6.0]
    - setuptools [required: >=19.2]
    - PyPrint [required: ~=0.2.3, installed: 0.2.3]
      - colorama [required: ~=0.3.6, installed: 0.3.7]
      - setuptools [required: >=19.2]
      - termcolor [required: ~=1.1.0, installed: 1.1.0]
    - libclang-py3 [required: ==0.2, installed: 0.2]
    - appdirs [required: ~=1.4.0, installed: 1.4.0]
  - cppclean [required: ~=0.9, installed: 0.9]
  - requests [required: ~=2.9.1, installed: 2.9.2]
  - restructuredtext-lint [required: ~=0.14.0, installed: 0.14.2]
    - docutils [required: >=0.11,<1.0, installed: 0.12]
  - html-linter [required: ~=0.3.0, installed: 0.3.0]
    - template-remover [installed: 0.1.9]
      - docopt [required: >=0.6.1, installed: 0.6.2]
    - docopt [required: >=0.6.1, installed: 0.6.2]
```